command/status.go

```diff
@@ -27,6 +27,9 @@ Usage: nomad status [options] <identifier>
   detect the type of resource being queried and display the appropriate
   status output.
 
+	If no arguments are provided, the command will fallback to "nomad job status",
+	which will list all jobs.
+
```

Comment:
It looks like you've got a tab (`\t`) here rather than space-aligned with the text above. An unfortunate quirk of writing big blocks of text in an editor configured to write Go code! :grinning: